LICENSE.md

```diff
@@ -1,6 +1,8 @@
 MIT License
 
-Copyright (c) 2019 Alexander Theimer
+Copyright (c) 2019 Alexander Theimer  
+Copyright (c) 2019-2021 twolaw  
```

Comment:
@twolaw do you have any other name to use here? do you want to be listed here?

Reply:
it looks good to me, thanks